programs/management/commands/import-catalog-data.py

```diff
@@ -274,24 +275,36 @@ def get_description(self, program_id):
 
         response = urllib2.urlopen(url)
         raw_data = response.read()
-        root = ET.fromstring(raw_data)
 
-        content = root.find('.//a:content', self.ns)
-
-        retval = ''
-
-        for el in content:
-            retval += ET.tostring(el, method='html')
-
-        retval = retval.replace('<h:', '<').replace('</h:', '</')
-
-        retval = retval.replace(' xmlns:h="http://www.w3.org/1999/xhtml"', '')
-
-        regex = re.compile(r'[\r\n\t]')
-
-        retval = regex.sub(' ', retval)
-
-        return retval
+        # Strip xmlns attributes to parse string to xml without namespaces
+        data = re.sub(' xmlns(?:\:[a-z]*)?="[^"]+"', '', raw_data)
+        root = BeautifulSoup(data, 'xml')
+        description_xml = root.find('content').encode_contents()
+        description_html = BeautifulSoup(description_xml, 'html.parser')
+
+        # Filter out invalid tags (replace them with span's)
```

Comment:
Comment is describing what will _not_ happen to the tags defined below it. Maybe move the comment down or update it to identify that "these tags will not be removed."